v0/destinations/fb_custom_audience/transform.js

```diff
@@ -66,7 +69,7 @@ function ensureApplicableFormat(userProperty, userInformation) {
     case "GEN":
       updatedProperty =
         userInformation.lowerCase() === "f" ||
-        userInformation.lowerCase() === "female"
+          userInformation.lowerCase() === "female"
```

Comment:
In any event, if any validations fail here the whole event will be dropped. So rather than directly throwing error for the whole batch of users. We can have a boolean flag returned from this function 
` validationPassed = ensureApllicableFormat()`

and let's keep a toggle in destination if the user enables validation button we will throw error for this flag false. Else we will just send the event without validation

WDYT?